tests/test_datastructures.py

```diff
@@ -0,0 +1,22 @@
+from connexion.datastructures import MediaTypeDict
+
+
+def test_media_type_dict():
```

Comment:
This file wasn't added to git yet in the PR where I introduced the MediaTypeDict type, so including it here.